UKHO.D365CallbackDistributorStub.API/UKHO.D365CallbackDistributorStub.API/Services/DistributionService.cs

```diff
@@ -41,5 +53,55 @@ public static bool SaveDistributorRequest(CustomCloudEvent cloudEvent)
                 return s_recordDistributorRequestQueue.ToList();
             }
         }
+
+        public bool SaveCommandDistributorRequest(string subject, HttpStatusCode? statusCode)
+        {
+            try
+            {
+                CommandDistributionRequest? commanddistributorRequest = s_CommandDistributionList.FirstOrDefault(a => a.Subject == subject);
```

Comment:
Variable name should be in Camel case please.
```suggestion
                CommandDistributionRequest? commandDistributorRequest = s_CommandDistributionList.FirstOrDefault(a => a.Subject == subject);
```

Reply:
Changes done